trino/sqlalchemy/datatype.py

```diff
@@ -86,6 +86,13 @@ def get_col_spec(self, **kw):
         return 'JSON'
 
 
+class UUID(UserDefinedType):
+    __visit_name__ = "UUID"
+
+    def get_col_spec(self, **kw):
+        return "UUID"
+
+
```

Comment:
I think this code block should be removed as we don't need to add a custom `UserDefinedType`